### ISSUE TITLE
Add "Articles" table from Pending Payments to individual "Payments" Dashboard

### ISSUE THREAD

**Issue by chucklapointe** [narcitymedia/lilium-cms#528]
**Is your feature request related to a problem? Please describe.**

Right now, contractors are not able to see details on how much they are getting paid per story in their payment dashboard. They can only see these details after it's been paid out and they receive an invoice via email every 2 weeks. 

If we are experiencing issues with Lilium payments and we expect the writers to alert their editors of any issues, it's basically impossible right now to do this before payments are sent out.
If we want to avoid any issues pre-emptively in the future, it would be better if contractors had access to this info before payments are made.

Right now, contractors can only see the amount of $ that is going to be paid out in their next payment, but no specifics on how much they will get paid in regards to certain pieces. Every two weeks, they get an email that shows specifics about payments per article in their invoice.

In the example with Colin, he wasn't able to identify where the issue was going to be on his next payment, he could only guess based on how much the dashboard said he was going to get paid that something was off. He suspected it was one article that was off, but when I looked at his dashboard, I was able to see that multiple stories were actually wrong. 

Not all writers are this detail-oriented, so I suspect that if there are other issues like this with payments, writers probably haven't caught them. The only people who will realize are people who meticulously look over invoices after getting paid and remember how much they were supposed to be paid per piece, or people like Colin who are good at looking at numbers and guessing there might be an issue with an upcoming payment.

**Describe the solution you'd like**

It would be super helpful if writers were able to see the same info we see in their payment dashboards so that they could make sure everything is accurate as they're writing stories throughout the month. If they spot any issues, they can let their editor know so we can fix it before it becomes a problem. I've attached a photo of my view of the payment dashboard (which includes details on what stories will be paid out in the contractor's next invoice). If we could make their dashboards like this, it would really help writers catch these problems!

If we weren't having any issues with payments, writers wouldn't need to see these details, but since we are, this seems like a good fix to make sure we catch any errors before these stories are paid out. 

In an ideal world, there wouldn't be any payment issues at all but this seems like a good fix for the meantime until we can make sure the system doesn't have any glitches.

![Screen Shot 2020-02-21 at 1.36.44 PM.png](https://images.zenhubusercontent.com/5e39effa9920ecebb862994b/172f6682-c283-4160-ba4c-329a2e84ad22)

![Screen Shot 2020-02-21 at 1.36.44 PM.png](https://images.zenhubusercontent.com/5e39effa9920ecebb862994b/bbafa82e-edfd-43fe-a329-0dded84762b4)

----

**TLDR**

Add a "Articles" table to their payments dashboard that lists pending articles to be paid, with the amounts. Similar to what is in the Pending Payments tab.

**Are you currently using a workaround? If so, please describe it.**

No. The errors are being spotted only 2 weeks after, and it's causing a lot of friction with the Finance and Editorial departments.

**How important do you think this feature is?**

Pretty important for Lilium satisfaction and employer reputation.
